### PR TITLE
Feature/1138 area of plants amount indicator

### PR DESCRIPTION
**Pull request by Bushuo**
## Basics

- [X] I added a line to [changelog.md](/doc/changelog.md)
- [X] The PR is rebased with current master.
- [X] Details of what you changed are in commit messages.
- [X] References to issues, e.g. `close #X`, are in the commit messages and changelog.
- [x] The buildserver is happy.

## Checklist

- [ ] I have installed and I am using [pre-commit hooks](../doc/contrib/README.md#Hooks)
- [ ] I fully described what my PR does in the documentation
- [ ] I fixed all affected documentation
- [ ] I fixed the introduction tour
- [ ] I wrote migrations in a way that they are compatible with already present data
- [ ] I fixed all affected decisions
- [ ] I added automated tests or a [manual test protocol](../doc/tests/manual/protocol.md)
- [ ] I added code comments, logging, and assertions as appropriate
- [ ] I translated all strings visible to the user
- [ ] I mentioned [every code or binary](https://github.com/ElektraInitiative/PermaplanT/blob/master/.reuse/dep5) not directly written or done by me in [reuse syntax](https://reuse.software/)
- [ ] I created left-over issues for things that are still to be done
- [ ] Code is conforming to [our Architecture](/doc/architecture)
- [ ] Code is conforming to [our Guidelines](/doc/guidelines)
- [ ] Code is consistent to [our Design Decisions](/doc/decisions)
- [ ] Exceptions to any guidelines are documented

## Review

- [ ] I've tested the code
- [ ] I've read through the whole code
- [ ] I've read through the whole documentation
- [ ] I've checked conformity to guidelines

### CODE REVIEW

**Comment by markus2330** [comment]
CI fails, would be great if it builds (maybe just a formatting problem), so that we can have a look on it.

**Comment by Bushuo** [comment]
Somehow those are files that I never touched 😂

File: frontend/src/config/env.ts [hunk 1]
```diff
@@ -7,3 +7,7 @@ export const nextcloudUri = import.meta.env.VITE_NEXTCLOUD_URI;
 
 export const isDev = import.meta.env.DEV;
 export const isProd = import.meta.env.PROD;
+
+// define this variable in a `.env.local` file,
+// queries and mutations will be executed even if no internet connection is available
```
Comment (markus2330) [inline frontend/src/config/env.ts:12]: Please write about such changes also in changelog.md

**Review by markus2330 — CHANGES_REQUESTED** [review]
Great job!

- [ ] Plants -> plants in the amount indicator
- [ ] the multiplication result of the plants would be useful, e.g. it should read ` 7 x 2 = 14 plants`
- [x] if you move the cursor to the very right, you don't see the plants amount indicator. I think it is difficult to fix and we don't think it is a problem though. (just a note)
- [x] the tour looks very differently, is this on purpose? (just a question)

**Comment by Bushuo** [comment]
@markus2330 

I checked in your change request.

> the tour looks very differently, is this on purpose?
Does the tour on master look different? Maybe it has to do with the changes I made to the tailwind config. If so I will take a look at it.

**Comment by Bushuo** [comment]
@markus2330 
I changed the prettier config for import sorting a little bit, I hope this is not a problem.
Eventual merge conflicts should be relatively easy to fix. Basically accept one of them and use auto import to sort it out.

The last commit therefore adds a bunch of changed files.

**Comment by markus2330** [comment]
> Does the tour on master look different?

It looked quite different on your PR when I tried it before but now I checked it again and Yvonne and me liked it very much. So it does not really matter if it changed or not. If it changed, it is now better. So everything a-okay :+1: 

Btw. in theory you can check on dev.permaplant.net how master looks like. For the tour it is a problem though if you already clicked it away... :wink: 

> I changed the prettier config for import sorting a little bit, I hope this is not a problem.

It is not a problem and actually I think this specific reformatting is a good idea. It would be better though if reformatting is done in separate PRs, as this makes reviewing much easier. (Reformatting PRs don't really need a review, except of the changed setting and changelog.md).

I hope @danielsteinkogler can nevertheless review it. Actually, it should be possible to filter out individual commits in a review.

**Comment by Bushuo** [comment]
@danielsteinkogler Yup, viewing only a range of commits in the review panel works.
Don't select the last 3 commits. 
I'm sorry 😅

**Comment by danielsteinkogler** [comment]
jenkins build please

**Comment by Bushuo** [comment]
I cherry picked the test fixes now into this branch to get this merged asap

**Comment by Bushuo** [comment]
jenkins build please

File: doc/changelog.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ Syntax: `- short text describing the change _(Your Name)_`
 - _()_
 - Migrate from Jest to Vitest, update Vite to v5, update Node to 20, .env should be .env.local _(Paul)_
 - Fix broken .env reading for scraper _(Paul)_
-- _()_
+- Add a dev env variable for react query when developing in offline env (e.g. plane) _(Paul)_
```
Comment (markus2330) [inline doc/changelog.md:37]: What is the name of this env var?
In general: Try to keep env vars as minimal as possible, only add them when you really need to change it from time to time.

**Review by markus2330 — APPROVED** [review]
Great job! :heart:

**Pull request merged**